db-client-java/src/main/java/com/eventstore/dbclient/CreateProjectionOptions.java

```diff
@@ -1,21 +1,21 @@
-package com.eventstore.dbclient;
-
-public class CreateContinuousProjectionOptions extends OptionsBase<CreateContinuousProjectionOptions> {
-    private boolean trackEmittedStreams;
-
-    private CreateContinuousProjectionOptions() {
-        this.trackEmittedStreams = false;
-    }
-
-    public static CreateContinuousProjectionOptions get() {
-        return new CreateContinuousProjectionOptions();
-    }
-
-    public boolean isTrackingEmittedStreams() {
-        return trackEmittedStreams;
-    }
-
-    public void trackEmittedStreams(boolean trackEmittedStreams) {
-        this.trackEmittedStreams = trackEmittedStreams;
-    }
-}
+package com.eventstore.dbclient;
+
+public class CreateProjectionOptions extends OptionsBase<CreateProjectionOptions> {
```

Comment:
**COMMENT:** `EmitEnabled` is missing, but my guess is that's because of the still lacking change to database proto, right? It'd be good to create an issue for that, not to forget adding that in the future.